examples/projections/nongeo/cartesian_linear.py

```diff
@@ -8,7 +8,7 @@
 
 fig = pygmt.Figure()
 fig.plot(
-    # The ``x`` and ``y`` arguments determine the coordinates of lines
+    # The ``x`` and ``y`` parameters determine the coordinates of lines
```

Comment:
```suggestion
    # The x and y parameters determine the coordinates of lines
```
Since these aren't rendered in ReST or markdown, I do not see the benefit of the code formatting here.